application/common/models/User.php

```diff
@@ -93,10 +94,6 @@ protected function attemptPasswordMigration()
         }
     }
 
-    /**
```

Comment:
Hmm... I guess that is redundant, looking at the code more closely now. My IDE shows more helpful info about a function if it has the doc. comments, but I don't know whether that's worth the (very minor) DRY violation. Either way seems fine, in this case.